doc/schemas/node_openapi.json

```diff
@@ -65,17 +65,23 @@
           "configs": {
             "$ref": "#/components/schemas/Configuration_array"
           },
+          "leadership_state": {
+            "$ref": "#/components/schemas/LeadershipState"
+          },
           "learners": {
             "$ref": "#/components/schemas/EntityId_to_uint64"
           },
-          "state": {
-            "$ref": "#/components/schemas/ReplicaState"
+          "membership_state": {
+            "$ref": "#/components/schemas/MembershipState"
+          },
+          "retirement_phase": {
+            "$ref": "#/components/schemas/RetirementPhase"
           }
         },
         "required": [
           "configs",
           "acks",
-          "state"
+          "membership_state"
```

Comment:
This looks like a breaking change, which should at least be mentioned in the changelog. Was the `state` field introduced after 1.x?

Reply:
@Amaury said so, but I haven't checked. I think the whole consensus details endpoint was introduced after 1.0 though.

Reply:
Not the right Amaury I guess...

Reply:
Sorry about that! I wanted @achamayou :-)